### PR TITLE
more documentation for README.md

### LINKED ISSUE #155: obsinfo is undocumented
With all the other work I've had in the last few months, I can't even remember what the obsinfo stuff is, let alone how it works.  This makes it really hard to review stuff like #138.  Please can you help my aching brain by writing one or two short paragraphs which explain it?  I think they should probably go in the `README.md`.

I admit that in general `tar_scm` could do with better documentation, but at least the `README.md` makes a half-hearted attempt to explain the standard functionality.  Unfortunately it got left behind when adding new functionality.

### PR DESCRIPTION
**Pull request by M0ses**
should fix #155 and #156 

* short description for each service
* documenation about archive formats

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,36 @@ This is the git repository for [openSUSE:Tools/obs-service-tar_scm](https://buil
 
 The service can be used in combination with other services like [download_files](https://github.com/openSUSE/obs-service-download_files), [extract_file](https://github.com/openSUSE/obs-service-extract_file), [recompress](https://github.com/openSUSE/obs-service-recompress) or [set_version](https://github.com/openSUSE/obs-service-set_version) e.g. within the [GIT integration](https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService#Example_2:_GIT_integration) workflow.
 
+## Archive Formats
+
+### obscpio
+The obscpio archive format is the new OBS specific archive format which used and generated by the [OBS Delta Store](http://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.architecture.html#delta_store) to store SCM changes in an incremental way. 
+It is used by the obs_scm service. A file <package>.obsinfo including useful information from your SCM system is generated, so you can access this data at buildtime.
```
Comment (aspiers) [inline README.md]: It's not 100% clear to me from this wording whether `obs_scm` *generates* obscpio files or whether it *consumes* them.  It seems to suggest that they are generated elsewhere (by the delta store) and "used by" (i.e. consumed) by `obs_scm`, but I think that would be a misinterpretation which contradicts the information on `obs_scm` below.  Any chance you could tweak the wording to clarify this?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,36 @@ This is the git repository for [openSUSE:Tools/obs-service-tar_scm](https://buil
...
+
+### tar
+The tar archive format is used to create a tarball and it is used by the services "tar" and "tar_scm"
```
Comment (aspiers) [inline README.md]: Similar thing here - if a file format is "used by" X, that could mean either X is generating files in that format *or* X is consuming files in that format which were generated by something else.

**Comment by M0ses** [comment]
@aspiers I hope the new version is a bit more clearer

**Comment by mmohring** [comment]
Another case of undocumented obsinfo leads to local builds with osc not working, because obsinfo is not copied by osc.

**Comment by aspiers** [comment]
@mmohring Not sure I follow entirely but please do submit follow-up issues (or even better, PRs)! Thanks!

**Comment by aspiers** [comment]
@mmohring Any chance you could submit a new issue for this?

**Pull request merged**